sqlx-postgres/src/types/bigdecimal.rs

```diff
@@ -154,7 +154,7 @@ impl Encode<'_, Postgres> for BigDecimal {
             })
```

Comment:
You can actually change this behavior to just return the error.

Reply:
Done!